system/modules/playos-status.nix

```
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }:
+let
+  ttyNumber = 8;
+  tty = "tty${toString ttyNumber}";
+  ttyPath = "/dev/${tty}";
+  driverUrl = "http://127.0.0.1:8382";
+
+  script =
+    pkgs.writeShellScriptBin "print-status" ''
+      while :; do
+        screen=$(xrandr --current | grep '*' | awk '{print $1}')
+        networkCount=$(connmanctl services | grep wifi | wc -l)
+        networkConnected=$(connmanctl services | grep wifi | grep "*" | awk '{ print $2 }')
+        rfid=$(curl -s "${driverUrl}/rfid/readers" | jq -r ".readers" | jq length)
+        controller=$(systemctl is-active playos-controller)
+        time=$(date +'%T %Z')
+        printf "\033c"
+        printf "%s\n" \
+          "Screen dimensions: $screen" \
+          "Wifi networks found: $networkCount" \
```

Vanity change :wink: 

```suggestion
          "Wi-Fi networks found: $networkCount" \
```

---

:+1: